test/Feature/Textures/SRVToUAV-array.test

```diff
@@ -0,0 +1,91 @@
+#--- source.hlsl
+
```

Comment:
Would be great to continue the pattern of describing the purpose of the test at the top here.